Tests/Resources/app/AppKernel.php

```diff
@@ -10,10 +10,16 @@ class AppKernel extends Kernel
      */
     public function registerBundles()
     {
-        return [
+        $bundles =  [
             new \Symfony\Bundle\FrameworkBundle\FrameworkBundle(),
             new \Http\HttplugBundle\HttplugBundle(),
         ];
+
+        if (false === defined('HHVM_VERSION')) {
+            array_splice($bundles, 1, 0, [new \Puli\SymfonyBundle\PuliBundle()]);
+        }
```

Comment:
Why do we need to insert the PuliBundle before HttplugBundle?
If the order is important we must document it.